docs/superpowers/plans/2026-07-31-per-item-content-filtering.md

```
@@ -0,0 +1,1461 @@
+                                Click the arrow next to a category to pick individual titles. Unticking a title
+                                stops it syncing and deletes its folder on the next sync.
+                            </div>
+```
+
+Find the equivalent series description (around line 900, above `.seriesCategoriesList`) with `sed -n '885,910p' Emby.Xtream.Plugin/Configuration/Web/config.html` and extend it the same way, substituting "series categories" / "series".
+
+- [ ] **Step 10: Build and deploy**
+
+```bash
+bash Emby.Xtream.Plugin/build.sh
+```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)